content/api/api-packages.md

```diff
@@ -8,199 +8,427 @@ alwaysopen: false
-                "dataSourceId": 0,
-                "id": "string",
-                "name": "string",
-                "value": "string"
-              }
-            ],
-            "fqnExcludes": [
-              "string"
-            ],
-            "fqnIncludes": [
-              "string"
-            ]
-          }
-        }
-      ],
-      "tags": [
-        {
-          "dataSourceId": 0,
-          "id": "string",
-          "name": "string",
-          "value": "string"
-        }
-      ],
-      "tenantId": "string",
-      "updated": "2018-06-26T19:39:40.742Z",
-      "userEmail": "string",
-      "version": "string"
+      "id": "f4a226ea-510f-3f27-8ca2-4e7211128728",
+      "tenantId": "0fa5384a-3b36-4766-ad68-43e7c6af54f6",
+      "packageId": "netuitive.packages.linux",
+      "version": "3.9.0",
+      "name": "Netuitive Packages Linux",
+      "description": "A set of Netuitive analytics configurations, polices, dashboards, and reports that are used to monitor performance of the default collectors in the Netuitive Agent (CPU, DiskSpace, DiskUsage, LoadAverage, Memory, Network, VMStat)",
+      "authorName": "Netuitive Research",
+      "authorEmail": "research@netuitive.com",
+      "logo": "http://assets.app.netuitive.com/pkg/logo/netuitive/banner_logo.png",
+      "downloadUrl": "https://github.com/netuitive-community-packages/netuitive-packages-linux/archive/master.zip",
+      "userEmail": null,
+      "created": "2020-02-27T22:37:04Z",
+      "updated": "2020-02-27T22:37:04Z",
+      "tags": [],
+      "policies": [],
+      "analyticConfigurations": [],
+      "dashboards": []
     }
   ]
 }
 ```
+{{% /expand %}}
+
+---
+
+## POST to /packages/install
+{{< button theme="success" href="https://app.metricly.com/swagger-ui.html#!/packages/installUsingPOST" >}} POST {{< /button >}} Use this endpoint to install custom packages as a multipart/form data zip file.
+
+{{% expand "View method details."%}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|-------------|----------------|-----------|----------------------|
+| User-Agent | header | string | User-Agent|
+| userEmail  | query | string  | Email address registered in CloudWisdom account. |
+| file  | formData  | file  |  A multipart/form data zip file. |
+
+### Request URL
+
+ `https://app.metricly.com/packages/install`
+
+{{% notice tip %}}
+
+CloudWisdom encourages that all custom packages be formatted, validated, and zipped using the metricly-cli: https://github.com/metricly/metricly-cli. Use these CLI commands in the following order: 1. `metricly package validate` 2. `metricly package format` 3. `metricly package create`.
+
+{{% /notice %}}
+
+### CURL
+
+The following example requires **Content-Type: multipart/form-data** be specified.
+
+```
+curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --header 'User-Agent: none' {"type":"formData"} 'https://app.metricly.com/packages/install'
```

Comment:
Forgot the `-F` flag for specifying the path to the local zip file that's getting posted.
```suggestion
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --header 'User-Agent: none' -F 'file=@./pkg.zip' {"type":"formData"} 'https://app.metricly.com/packages/install''
```